doc/developer-guide/api/functions/TSRemap.en.rst

```diff
@@ -69,6 +69,11 @@ the HTTP transaction.
 remap configuration file (:file:`remap.config`) is reloaded. This is an optional
 entry point, which takes no arguments and has no return value.
 
+Generally speaking, calls to these functions are mutually exclusive. The exception
+is for functions which take an HTTP transaction as a parameter. Calls to these
```

Comment:
I would call out TSRemapDoRemap() and TSRemapOSResponse(), but it is up to you.

For example:
functions which take an HTTP transaction as a parameter: TSRemapDoRemap() and TSRemapOSResponse().